packages/controller/src/UserManager.ts

```diff
@@ -72,11 +72,9 @@ export default class UserManager {
 		}
 
 		let defaultRoleId = this._config.get("controller.default_role_id");
-		if (defaultRoleId === null) {
-			throw new Error(`controller.default_role_id is not defined.`);
-		}
+		let roles = defaultRoleId ? [defaultRoleId] : [];
```

Comment:
Note that defaultRoleId can be set to 0, which is the default admin role. If you intended for that value to be ignored you should add a comment saying so, otherwise compare it with null.